ts/Series/Timeline/TimelineSeries.ts

```diff
@@ -110,8 +110,10 @@ class TimelineSeries extends LineSeries {
     public alignDataLabel(
         point: TimelinePoint,
         dataLabel: SVGLabel,
+        /* eslint-disable @typescript-eslint/no-unused-vars */
         _options: TimelineDataLabelOptions,
         _alignTo: BBoxObject
+        /* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
See the other comment about https://eslint.org/docs/latest/rules/no-unused-vars#varsignorepattern.